### DISCUSSION TITLE
ts js mappings for group filtering

### DISCUSSION THREAD

**Discussion by Diaskhan** [alirezanet/Gridify#77] [Ideas]
Hie peoople.

I have few questions about typescript and js.

1.Is there some ts js helpers like this 
https://github.com/nestjsx/crud/wiki/Requests#frontend-usage ?

It could help make type specific filters on Frontend.
https://primefaces.org/primeng/table/filter

![image](https://user-images.githubusercontent.com/15046677/164328252-0520806f-eeed-445e-8abe-58d18b248804.png)

**Comment by alirezanet** [comment]
Hi @Diaskhan, 
The main use case of gridify is exactly like this, but gridify is a dotnet tool that you can integrate into any `js` `ts` and front-end library although It doesn't have any built-in front-end tool by itself.

**Reply by alirezanet** [reply]
I think having some front-end tools like this example, specifically for gridify could be really nice but as a separate project.

**Comment by ds1371dani** [comment]
Hi guys,
I recently had to implement filters using this package in `react`, so I created a gist from my usage.
Here it is:
https://gist.github.com/ds1371dani/a03461aa0dfb2330afa718f07a26a47a

**Reply by alirezanet** [reply]
Great, Thank you for sharing the code 💐

**Comment by alirezanet** [comment]
Just a quick update on the JS library,
I just released a preview version. some early feedback would be really nice :) , 

https://www.npmjs.com/package/gridify-client

more info:
https://github.com/alirezanet/Gridify/pull/136
